lib/filewatch/watch.rb

```diff
@@ -161,6 +184,23 @@ def _discover_file(path, initial=false)
       end
     end # def _discover_file
 
+    private
+    def _globbed_files(path)
+      globbed_dirs = Dir.glob(path)
+      @logger.debug? && @logger.debug("_globbed_files: #{path}: glob is: #{globbed_dirs}")
+      if globbed_dirs.empty? && File.file?(path)
+        globbed_dirs = [path]
+        @logger.debug? && @logger.debug("_globbed_files: #{path}: glob is: #{globbed_dirs} because glob did not work")
+      end
+      # return Enumerator
+      globbed_dirs.to_enum
```

Comment:
Because I did not want to return the array that can be abused by future code only an enumerator so future code can only iterate over it.